web/packages/teleterm/src/ui/ThemeProvider/theme.ts

```diff
@@ -31,12 +31,12 @@ const colors = {
     surface: '#222C59',
     surfaceSecondary: '#182047',
```

Comment:
@rudream What's the story with those `levels.*Secondary` colors, did they get removed from the Web UI themes at some point? @gzdunek We should probably do the same thing that the Web UI did with them.

Reply:
Yes, I removed them since they were not part of the figma design system.